crates/ruff_python_formatter/src/pattern/pattern_arguments.rs

```diff
@@ -0,0 +1,111 @@
+use ruff_formatter::write;
+use ruff_python_ast::node::AstNode;
+use ruff_python_ast::{Pattern, PatternArguments, Ranged};
+use ruff_python_trivia::{SimpleTokenKind, SimpleTokenizer};
+use ruff_text_size::{TextRange, TextSize};
+
+use crate::comments::SourceComment;
+use crate::expression::parentheses::{empty_parenthesized, parenthesized, Parentheses};
+use crate::prelude::*;
+use crate::FormatNodeRule;
+
+#[derive(Default)]
+pub struct FormatPatternArguments;
```

Comment:
This is very, very similar to `arguments.rs`, but it's difficult to share much code.